gluten-ut/spark32/src/test/scala/org/apache/spark/sql/execution/datasources/parquet/GlutenParquetFilterSuite.scala

```diff
@@ -301,7 +301,7 @@ abstract class GltuenParquetFilterSuite extends ParquetFilterSuite with GlutenSQ
   }
 }
 
-class GlutenParquetV1FilterSuite extends ParquetV1FilterSuite with GlutenSQLTestsBaseTrait {
+class GlutenParquetV1FilterSuite extends GltuenParquetFilterSuite with GlutenSQLTestsBaseTrait {
```

Comment:
Could you please also fix `GlutenParquetV2FilterSuite` in the same way?

Reply:
Fixed. Thanks!